protocols/virtualdefund/virtualdefund.go

```diff
@@ -21,7 +21,7 @@ const (
 )
 
 // The turn number used for the final state
-const FinalTurnNum = 2
+const FinalTurnNum = 3
```

Comment:
Our consensus_channel [logic](https://github.com/statechannels/go-nitro/blob/1b4e0577a07b8d3944f75a4469de6a50621d4b59/channel/consensus_channel/consensus_channel.go#L751)  assumes that a proposal will have a turn number **after** the current consensus turn.

By using a hardcoded value of 3 this works for the simple case of a ledger channel funding one virtual channel, but this will break down with more payments. We should not be using a specific turn number when we're generating/ accepting proposals

Reply:
This has been addressed in #620